demos/wait-for-pods-ready/README.md

```diff
@@ -0,0 +1,5 @@
+Kueue Gang Scheduling and WaitForPodsReady Feature Demo
+
+DRAFT
+
+[Demo recording](https://asciinema.org/a/709849) (hosted)
```

Comment:
It would be nice to see workload statuses during this demo showcasing the eviction message for the failed job

Reply:
Updated, thanks for the great idea!